app/src/main/java/com/example/testoauth/ui/login/SignInActivity.java

```diff
@@ -151,6 +153,25 @@ public void onRequestSuccess(final int requestCode, @Nullable VerificationDataBu
                         Toast.LENGTH_SHORT).show();
                 showLayout(PROFILE_LAYOUT);
                 findViewById(R.id.btnVerify).setOnClickListener(verifyClickListener);
+            } else if (requestCode == VerificationCallback.TYPE_IM_OTP_INITIATED) {
+                verificationCallbackType = VerificationCallback.TYPE_IM_OTP_INITIATED;
+                String ttl = bundle.getString(VerificationDataBundle.KEY_TTL);
+                if (ttl != null) {
+                    Toast.makeText(SignInActivity.this,
```

Comment:
toast is used multiple times in this class, helper extension function can be created

Reply:
Yes, makes sense, but since this is a sample app to demonstrate how to integrate our SDK, we are not concerned about creating helper extension functions.